localstack-core/localstack/aws/client.py

```diff
@@ -243,9 +243,11 @@ def _patched_encode_datetime(self, value: datetime) -> None:
             else:
                 timestamp = timegm(value.utctimetuple()) + value.microsecond / 1000000
             # The next line is the only change in this patch compared to the original function.
-            # AWS breaks the CBOR spec by using the millis instead of seconds (with floating point support for millis)
-            # https://github.com/aws/aws-sdk-java-v2/issues/4661
-            timestamp = timestamp * 1000
+            # - AWS breaks the CBOR spec by using the millis instead of seconds (with floating point support for millis)
```

Comment:
Kind of, the brackets were set a bit misleading. I changed the line to
"AWS breaks the CBOR spec by using the millis (instead of seconds with floating point support for millis)"
😅

Reply:
Ah thanks, I understand 😄